### ISSUE TITLE
[KIEKER-1715] Text reader has trouble reading empty values

### ISSUE THREAD

**Issue by rju** [kieker-monitoring/kieker#2124]
JIRA Issue: [KIEKER-1715] Text reader has trouble reading empty values
Original Reporter: Reiner Jung
 *** 
<p>The following line was produced by the `AsciiFileWriter`:<br/>
`$1;1521828690426151555;-6758008612213227520;15;&lt;no-session-id&gt;;31502c6809c3;-6758008612213227520;-1;`<br/>
As you can see the last value is empty. This causes and out of bounds exception. As the trailing empty field treated correctly.</p>

[KIEKER-1715]: https://kieker-monitoring.atlassian.net/browse/KIEKER-1715?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by rju** [comment]
author rju -- Wed, 28 Mar 2018 15:09:08 +0200
<p>I fixed the TextValueDeserializer which is now able to read these files correctly. Also we developed a new file read in iObserve which utilizes the TextValueDeserializer and not the array-API of Kieker which is deprecated. Shall we move the iObserve reader to Kieker and replace the existing `DatFile2RecordFilter`? However, the iObserve reader does no longer support Kieker 1.5 OperationExecutionRecords.</p>
